src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -694,6 +694,31 @@ static int _write_script_header_to_fd(struct xccdf_policy *policy, struct xccdf_
 		const struct xccdf_version_info *xccdf_version = xccdf_benchmark_get_schema_version(benchmark);
 		const char *xccdf_version_name = xccdf_version_info_get_version(xccdf_version);
 
+		if (NULL != profile_description) {
+			unsigned int newLines = 0;
```

Comment:
code style

Reply:
You mean the NULL coming first? I can change it, but it's a technique to avoid typing **var = NULL** instead **var == NULL**.